docs/examples/use_cases/tensorflow/yolov4/src/dali/ops.py

```diff
@@ -0,0 +1,212 @@
+import nvidia.dali as dali
+
+def input(file_root, annotations_file, device_id, num_threads, device, random_shuffle=True):
```

Comment:
```suggestion
def input(file_root, annotations_file, shard_id, num_threads, device, random_shuffle=True):
```

Reply:
In case of single node it is the same, but not for multinode. I would prefer to keep the naming right,

Reply:
Changed